examples/Console/TestZipkinExporter.cs

```diff
@@ -23,6 +23,9 @@ internal static object Run(string zipkinUri)
 
         // Enable OpenTelemetry for the sources "Samples.SampleServer" and "Samples.SampleClient"
         // and use the Zipkin exporter.
+
+        zipkinUri ??= "http://localhost:9411/api/v2/spans";
```

Comment:
This is the value from the comment on line 22. 
I don't know if this is an actual server default value or just an example for the comment above.

If this isn't an actual default value, then I think it would be more appropriate to throw an ArgumentNullException.